From 0a9d9ab4949effb20e77aa52e7b4ee07e776fc0d Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 4 Sep 2018 09:58:21 +0200
Subject: daemon: Fix upower's keyboard backlight support

Disabling kernel tunables blocked us from accessing /sys, meaning we
couldn't change keyboard backlight levels.

Closes: #73
---
 src/upower.service.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/upower.service.in b/src/upower.service.in
index 8838655..4566ad3 100644
--- a/src/upower.service.in
+++ b/src/upower.service.in
@@ -10,7 +10,8 @@ Restart=on-failure
 
 # Filesystem lockdown
 ProtectSystem=strict
-ProtectKernelTunables=true
+# Needed by keyboard backlight support
+ProtectKernelTunables=false
 ProtectControlGroups=true
 ReadWritePaths=@historydir@
 ProtectHome=true
-- 
cgit v1.1

